cia.mk

```diff
@@ -1,59 +1,91 @@
 ### Virtual Console repacking stuff
 
-# Include Configuration Settings
 vc_dir := poke-cia
--include $(vc_dir)/cia-config.mk
+
+# Include Configuration Settings
+include ${vc_dir}/cia-config.mk
+ifeq ($(strip ${vc_name}),)
+$(error Please set the `vc_name` variable in ${vc_dir}/cia-config.mk)
+endif
+
+vc_rom_dirs   := $(addprefix ${vc_dir}/, ${vc_name})
+vc_cias       := $(addsuffix .cia, ${vc_rom_dirs})
+vc_orig_cia   := $(addsuffix .orig.cia, ${vc_rom_dirs})
+vc_game_cxi   := $(addsuffix .game.cxi, ${vc_rom_dirs})
+vc_manual_cfa := $(addsuffix .manual.cfa, ${vc_rom_dirs})
+
+
+# "Interface" rules
 
 .PHONY: cia
-cia: $(vc_cia)
+cia: ${vc_cias}
 
 .PHONY: clean
 clean: clean_cia
 
 .PHONY: clean_cia
 clean_cia:
-	rm -f $(vc_cia) $(vc_game_cxi) $(vc_manual_cfa)
+	rm -f ${vc_cias} ${vc_game_cxi} ${vc_manual_cfa}
 
 .PHONY: distclean_cia
 distclean_cia: clean_cia
-	rm -rf $(vc_rom_dir) $(vc_dir)/seeddb.bin
+	rm -rf ${vc_rom_dirs} ${vc_dir}/seeddb.bin
+
 
-$(vc_rom_dir): $(vc_orig_cia) $(vc_dir)/seeddb.bin
+# Actual rules
+
+# TODO: relying on directory modification time is a bad idea!
+# Silence `ctrtool`, which is VERY verbose by default
+${vc_dir}/%/: ${vc_dir}/%.orig.cia ${vc_dir}/seeddb.bin
```

Comment:
Not sure why the wildcard directory target is not working for me... probably more reason to move away from directory targets.

``` Considering target file 'poke-cia/pokegold/'.
     File 'poke-cia/pokegold/' does not exist.
     Finished prerequisites of target file 'poke-cia/pokegold/'.
    Must remake target 'poke-cia/pokegold/'.
make: *** No rule to make target 'poke-cia/pokegold/', needed by 'poke-cia/pokegold/romfs/pokegold.patch'.  Stop.
```

Reply:
Once I created the directories once manually.. the error went away... Even if I deleted them later... super weird.